### DISCUSSION TITLE
Fresh teslamate setup on Synology is not working

### DISCUSSION THREAD

**Discussion by sjayanna01** [teslamate-org/teslamate#3285] [Q&A]
Hello Teslamate users,

I have installed Teslamate on Synology DS920+, following the [guide](https://github.com/adriankumpf/teslamate/issues/113).
Docker app is not supported, so installed it on Container Manager - version 20.10.23-1413

Grafana works fine: 192.xxx.xxx.xxx:3000
Teslamate doesn't load: 192.xxx.xxx.xxx:4000

Teslamate logs - Waiting for postgres at db:5432.
Attached my docker compose file for reference.

I did see a thread with similar issue but couldn't find what exactly was done to resolve the issue.

Thanks in advance!

UPDATE: Attached teslamate and database container logs.

[docker-compose copy.txt](https://github.com/adriankumpf/teslamate/files/12073669/docker-compose.copy.txt)
[docker-database-1.csv](https://github.com/adriankumpf/teslamate/files/12074006/docker-database-1.csv)
[docker-teslamate-1.csv](https://github.com/adriankumpf/teslamate/files/12074007/docker-teslamate-1.csv)

**Comment by sjayanna01** [comment]
Resolved!

Removed the containers, uninstalled and reinstalled the Container Manager.
Changed the location of directories in the composer yml file before.

Everything started working fine!

**Reply by windy1lee** [reply]
Hello, I'm currently facing the same issue as you. I've tried installing Teslamate on my NAS several times, and the problem is consistently the same - port 3000 opens, but port 4000 stops working. Could you please share your docker-compose.yml file with me? Thank you.

**Reply by tempod** [reply]
same error here. how did you solve it?

**Reply by windy1lee** [reply]
> 这里也有同样的错误。你是怎么解决的？

It's been too long, I don't quite remember. It should be an issue with the NAS firewall. I turned off the firewall and tried again, so I reset the firewall!
